DashKitUI/ViewControllers/DashSettingsViewController.swift

```diff
@@ -59,11 +59,13 @@ class DashSettingsViewController: UITableViewController {
     }
 
     private func done() {
-        if let nav = navigationController as? SettingsNavigationViewController {
+        switch navigationController {
```

Comment:
💯Clever! I like this. I'll have to remember this pattern when in a similar situation.